src/leaderboardBot.py

```diff
@@ -304,3 +302,22 @@ def clearDailyTable(self):
                     }
                 )
 
+    def addDefaultAlias(self):
+        for key in default_alias:
+            item = {
+                'Alias': key,
+                'PlayerName': default_alias[key]
+            }
+            self.alias_table.put_item(Item=item)
+
+    def addChannels(self):
+        for key in default_channels:
+            item = {
+                'ChannelName': key
```

Comment:
need `'PlayerName'` in item

Reply:
or you could add all the channels as aliases also

Reply:
> need `'PlayerName'` in item

Good call, I think no argument commands aren't supported right now